lib/modules/settingsConsole.js

```diff
@@ -799,6 +829,10 @@ function stageCurrentModuleOptions() {
 							const tempArray = input.value.split(',');
 							// convert the internal values of this array into their respective types (int, bool, bool, bool)
 							optionValue = [parseInt(tempArray[0], 10), (tempArray[1] === 'true'), (tempArray[2] === 'true'), (tempArray[3] === 'true')];
+						} else if (input instanceof HTMLSelectElement) {
+							if (input.selectedIndex >= 0) {
```

Comment:
`optionValue = input.value` can be used

Reply:
Should I keep that `else if` for the sake of readability, or remove it since the next `else` is `optionValue = input.value` anyways?

Reply:
Doesn't seem necessary to keep it, as the final `else` is clear enough.